src/sentry/web/forms/accounts.py

```diff
@@ -163,6 +163,16 @@ class RegistrationForm(forms.ModelForm):
         widget=forms.TextInput(attrs={'placeholder': 'you@example.com'}))
     password = forms.CharField(
         widget=forms.PasswordInput(attrs={'placeholder': 'something super secret'}))
+    subscribe = forms.BooleanField(
+        label=_('Subscribe to product updates newsletter'),
```

Comment:
Suggestion on copy here is welcome, but this only affects onpremise, since we use a different registration flow on sentry.io